tests/_support/Step/Acceptance/Administrator.php

```diff
@@ -0,0 +1,40 @@
+<?php
```

Comment:
:no_entry_sign: **Error**: Class file names should be based on the class name with &quot;class-&quot; prepended. Expected class-administrator.php, but found vipgoci-phpcs-scan-D2OKzW.php (*WordPress.Files.FileName.InvalidClassFileName*).